packages/api-client/src/main/user/UserAPI.ts

```diff
@@ -38,6 +39,7 @@ import {
 } from '../user/';
 
 export class UserAPI {
+  static DEFAULT_USERS_CHUNK_SIZE = 50;
```

Comment:
```suggestion
  static readonly DEFAULT_USERS_CHUNK_SIZE = 50;
```

Or will we change this value?